contest-api/check-api.sh

```diff
@@ -161,6 +165,7 @@ if [ -z "$API_URL" ]; then
 fi
 
 TMP=$(mktemp -d)
+echo $TMP
```

Comment:
I assume this is a debugging leftover? If not, then use `verbose` to print it, so that it can be silenced.

Reply:
Lol yes my bad.